tests/integration/helpers/cluster.py

```diff
@@ -2525,19 +2526,20 @@ def get_kazoo_client(self, zoo_instance_name):
     def run_kazoo_commands_with_retries(
         self, kazoo_callback, zoo_instance_name="zoo1", repeats=1, sleep_for=1
     ):
-        zk = self.get_kazoo_client(zoo_instance_name)
```

Comment:
Why not create this client in advance and reuse it in test? We can leave it connected after we check that zookeeper is working